operators/multiclusterobservability/controllers/multiclusterobservability/analytics/rs-virtualization/helper.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+// Licensed under the Apache License 2.0
+
+package rsvirtualization
+
+import (
+	"context"
+
+	mcov1beta2 "github.com/stolostron/multicluster-observability-operator/operators/multiclusterobservability/api/v1beta2"
+	rsutility "github.com/stolostron/multicluster-observability-operator/operators/multiclusterobservability/controllers/multiclusterobservability/analytics/rs-utility"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	// Virtualization-specific resource names
+	PlacementBindingName     = "rs-virt-policyset-binding"
+	PlacementName            = "rs-virt-placement"
+	PrometheusRulePolicyName = "rs-virt-prom-rules-policy"
+	PrometheusRuleName       = "acm-rs-virt-prometheus-rules"
+	ConfigMapName            = "rs-virt-config"
+)
+
+var (
+	log = logf.Log.WithName("rs-virtualization")
+
+	// Component configuration
+	componentConfig = rsutility.ComponentConfig{
+		ComponentType:            rsutility.ComponentTypeVirtualization,
+		ConfigMapName:            ConfigMapName,
+		PlacementName:            PlacementName,
+		PlacementBindingName:     PlacementBindingName,
+		PrometheusRulePolicyName: PrometheusRulePolicyName,
+		DefaultNamespace:         rsutility.DefaultNamespace,
+		GetDefaultConfigFunc:     GetDefaultRSVirtualizationConfig,
+		ApplyChangesFunc:         ApplyRSVirtualizationConfigMapChanges,
+	}
+
+	// Component state
+	ComponentState = &rsutility.ComponentState{
+		Namespace: rsutility.DefaultNamespace,
+		Enabled:   false,
+	}
```

Comment:
Just an idea. What about encapsulating this in some new type and changing the functions using this var to methods for that type? The new type could probably have the client too.